lib/job/arObjectMoveJob.class.php

```diff
@@ -120,4 +122,16 @@ public function runJob($parameters)
 
         return true;
     }
+
+    public function reindexObjectRange(array $children, int $start, int $end)
+    {
+        for ($i = $start; $i <= $end; ++$i) {
+            $objectId = $children[$i]['id'];
+            $this->info($this->i18n->__('Reindexing object id: %1', ['%1' => $objectId]));
```

Comment:
I think it's better to move this into the conditional, so it only logs for ids that actually get reindexed.

Reply:
Makes sense - Fixed.